doc/source/user_guide/hpc/pymapdl.rst

```diff
@@ -1,205 +1,169 @@
-  This normally implies that the ``ANSYS`` installation directory is in a
-  shared drive or directory. Your HPC cluster administrator
-  should provide you with the path to the ``ANSYS`` directory.
+Introduction
+============
 
-* **A compatible Python installation must be accessible from all the compute nodes**.
-  For compatible Python versions, see :ref:`ref_pymapdl_installation`.
+PyMAPDL communicates with MAPDL using the gRPC protocol.
+This protocol offers many advantages and features, for more information
+see :ref:`ref_project_page`.
+One of these features is that it is not required to have both,
+PyMAPDL and MAPDL processes, running on the same machine.
+This possibility open the door to many configurations, depending
+on whether you run them both or not on the HPC compute nodes.
+Additionally, you might to be able interact with them (``interactive`` mode)
+or not (``batch`` mode).
 
-Additionally, you must perform a few key steps to ensure efficient job
-execution and resource utilization. Subsequent topics describe these steps.
+Currently, the supported configurations are:
 
-Check the Python installation
------------------------------
+* :ref:`ref_pymapdl_batch_in_cluster_hpc`
 
-The PyMAPDL Python package (``ansys-mapdl-core``) must be installed in a virtual
-environment that is accessible from the compute nodes.
 
-To see where your Python distribution is installed, use this code:
+Since v0.68.5, PyMAPDL can take advantage of the tight integration
+between the scheduler and MAPDL to read the job configuration and
+launch an MAPDL instance that can use all the resources allocated
+to that job.
+For instance, if a SLURM job has allocated 8 nodes with 4 cores each,
+then PyMAPDL launches an MAPDL instance which uses 32 cores
+spawning across those 8 nodes.
+This behaviour can turn off if passing the environment variable
+:envvar:`PYMAPDL_ON_SLURM` or passing the argument `detect_HPC=False`
+to :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`.
 
-.. code-block:: console
 
-    user@machine:~$ which python3
-    /usr/bin/python3
+.. _ref_pymapdl_batch_in_cluster_hpc:
 
-To print the version of Python you have available, use this code:
+Submit a PyMAPDL batch job to the cluster from the entrypoint node
+==================================================================
 
-.. code-block:: console
+Many HPC clusters allow their users to login in a machine using
+``ssh``, ``vnc``, ``rdp``, or similar technologies and submit a job
+to the cluster from there.
+This entrypoint machine, sometimes known as *head node* or *entrypoint node*,
+might be a virtual machine (VDI/VM).
 
-    user@machine:~$ python3 --version
-    Python 3.9.16
+In such cases, once the Python virtual environment with PyMAPDL is already
+set and is accessible to all the compute nodes, launching a
+PyMAPDL job is very easy to do using ``sbatch`` command.
+No changes are needed on a PyMAPDL script to run it on an SLURM cluster.
```

Comment:
Is mapdl server running on the head node or compute node in this case? I assume it is the compute node, but if so it should say this clearly here

Reply:
Got it.